generators/python/src/fern_python/generators/sdk/client_generator/websocket_connect_method_generator.py

```diff
@@ -615,7 +619,7 @@ def write_ternary(writer: AST.NodeWriter) -> None:
             context=self._context, object_=reference, type_reference=query_parameter.value_type
         )
 
-    def _get_environment_as_str(self, *, websocket: ir_types.WebSocketChannel) -> str:
+    def _get_environment_as_str(self, *, websocket: ir_types.WebSocketChannel) -> Optional[str]:
```

Comment:
nit: can we rename this method to `_get_ multiple_base_url_environment_as_string`